src/app/utils/usePostSet.tsx

```diff
@@ -0,0 +1,17 @@
+import { SetProps } from "./useSet";
+
+export default function usePostSet(set: SetProps | null) {
```

Comment:
This is not a hook, so the naming should be different :) 
A custom hook is only a "Hook" when it uses React Hooks (like useState or useEffect, etc.).

```suggestion
export default function postSet(set: SetProps | null) {
```